docs/source/configuration/volto-slate/index.md

```diff
@@ -11,7 +11,7 @@ myst:
 
 # `volto-slate`
 
-`volto-slate` is an interactive default text editor for Volto, developed on top of {term}`Slate`.
+`volto-slate` is an interactive default text editor for Volto, integrated into the core system and developed on top of {term}`Slate`.
```

Comment:
I think the more important clause should go first, although the lesser is a good addition for clarity.

```suggestion
`volto-slate` is an interactive default text editor for Volto, developed on top of {term}`Slate` and integrated into the core system.
```